network-api/networkapi/wagtailpages/pagemodels/buyersguide/editorial_content_index.py

```diff
@@ -22,6 +23,14 @@ class BuyersGuideEditorialContentIndexPage(
     subpage_types = ['wagtailpages.BuyersGuideArticlePage']
     template = 'pages/buyersguide/editorial_content_index_page.html'
 
+    call_to_action = models.ForeignKey(
+          'wagtailpages.BuyersGuideCallToAction',
+          null=True,
+          blank=True,
+          on_delete=models.SET_NULL,
+          related_name='+'
+      )
+
```

Comment:
**Issue** The content index page should display the "featured CTA". The CTA that is set on the buyer's guide homepage is the featured CTA. There is no need for this field. You would rather need a method that returns the featured CTA.